partitura/io/matchfile_utils.py

```diff
@@ -871,6 +871,47 @@ def format_time_signature_list(value: MatchTimeSignature) -> str:
     return str(value)
 
 
+class MatchTempoIndication(MatchParameter):
+    def __init__(
+            self,
+            value: str,
+            is_list: bool = False,
+    ):
+        super().__init__()
+        self.value = self.from_string(value)[0]
+        self.is_list = is_list
+    
+    def __str__(self):
+        return self.value
+
+    @classmethod
+    
+    def from_string(cls, string: str) -> MatchTempoIndication:
+        
+        # NOTE particularities of the BpM dataset....
+        if string is not None:
+            if 'Rond' in string:
+                content = string.split(' ')
+                content = [content[-1]]
+            elif 'Alla' in string: # for kv331_3
```

Comment:
This seems to be a very particular case, and can lead to other issues. Maybe we can have a list of equivalences somewhere that covers these cases?